addons/website/static/src/scss/website.scss

```diff
@@ -200,6 +200,12 @@ font[class*='bg-'] {
         border-bottom-right-radius: $size;
     }
 }
+// Smaller container
+$o-container-small-width: 620px;
```

Comment:
Put the variable definition in website/primary_variables.scss and add the `!default` flag. This will allow to override the variable in an extension/theme and to use that variable in other assets bundle if necessary.